src/OrchardCore.Modules/OrchardCore.Users/Views/Items/ValidateUserTask.Fields.Thumbnail.cshtml

```diff
@@ -0,0 +1,2 @@
+<h4 class="card-title"><i class="fa fa-user"></i>@T["Validate User"]</h4>
+<p>@T["Check if the user exists for the current request and has the specified role(s)."]</p>
```

Comment:
@agriffard can you suggest wording for this ?

Reply:
I don't know how to fomulate it otherwise.